service/plugin_service.go

```diff
@@ -93,13 +93,17 @@ func (d DependencyPluginService) compileConfig(ctx context.Context, configs mode
 		compiler.From(secrets.ToMap()).WithName("secret"),
 	)
 
-	for i, config := range configs {
+	var confs models.PluginConfigs
+	for _, config := range configs {
 		compiledCnf, err := d.engine.CompileString(config.Value, tmplCtx)
 		if err == nil {
```

Comment:
why are we ignoring the error, either we should fail or we need to add the config before compilation, in this situatin I believe we can throw an error

Reply:
We will add the config before the error